widget/embedded/src/components/ConfirmWalletsModal/ConfirmWalletsModal.tsx

```diff
@@ -138,6 +146,37 @@ export function ConfirmWalletsModal(props: PropTypes) {
     });
   };
 
+  const handlePasteCustomDestination = async (
+    event: MouseEvent<HTMLButtonElement>
+  ) => {
+    event.preventDefault();
+    if (navigator.clipboard !== undefined) {
+      const pastedText = await navigator.clipboard.readText();
+      setCustomDestination(pastedText);
+      customDestinationInputRef?.current?.focus();
```

Comment:
Currently, if there is no text in the clipboard, we show the paste icon. Can we modify it to display the paste icon only if there is data in the clipboard?

Reply:
I don't think that this is necessary. I have not seen anything similar to what you say.